# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/base/BodyStream.h
# Commit: acb4e003bfae
# Full Hash: acb4e003bfae8bf1e90a6a9d059a995ec1c72239
# Author: Kagami Sascha Rosylight <krosylight@mozilla.com>
# Date: 2023-02-18 04:50:51
# Regressor Bug: 1813011
# File Overlap Count: 1
# Description:
#   Bug 1813011 - Part 1: Do not call AsyncAwait again after each pull r=smaug
#   
#   The reader will only get the close signal by sending another read request, so no need to check it greedily.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D169482
# ==============================================================================

diff -r 5505e602d7cf -r acb4e003bfae dom/base/BodyStream.h
--- a/dom/base/BodyStream.h	Fri Feb 17 18:32:24 2023 +0000
+++ b/dom/base/BodyStream.h	Fri Feb 17 18:32:24 2023 +0000
@@ -150,11 +150,6 @@
     // We are ready to write something in the JS Buffer.
     eWriting,
 
-    // After a writing, we want to check if the stream is closed. After the
-    // check, we go back to eWaiting. If a reading request happens in the
-    // meantime, we move to eReading state.
-    eChecking,
-
     // Operation completed.
     eClosed,
   };
